### ISSUE TITLE
Success Criteria Emails Don't Indicate Which WCAG Versions They Apply To

### ISSUE THREAD

**Issue by Grunet** [Grunet/digestible-wcag#1]
The use case here is for someone who wants to easily tell if the email is relevant for their current work or not (e.g. they might be working at a company where the compliance target is WCAG 2.1, and feel misled if they get an email from a success criterion that was only introduced in WCAG 2.2, thinking that they need to address it in their work)

(WCAG version referring to the "2.0", "2.1", "2.2.", etc... suffix)

**Comment by Grunet** [comment]
This will likely need changes in the digestible-wcag-scraping repository to scrape that information off of the WCAG pages, changes in the digestible-wcag-templates repository to design out how the info should be represented in the emails, and lastly changes in the digestible-wcag-sc-emails repository to put those 2 changes together in the generated emails.

**Comment by Grunet** [comment]
This issue (https://github.com/w3c/wcag/issues/1421) in the Aug 20 version of WCAG 2.2 may be a partial blocker to getting every last success criteria email updated, but it should still be possible to get most of the data using the 2.1 versions of the pages linked out from the Feb 20 version of WCAG 2.2
